packages/evm/src/interpreter.ts

```diff
@@ -272,9 +271,7 @@ export class Interpreter {
         const contract = this._runState.interpreter.getAddress()
 
         if (
-          !(await (
-            this._runState.stateManager as StatelessVerkleStateManager
```

Comment:
This generally won't affect tree shaking. TypeScript types are always removed even if they are values or classes as long as they are used as a type as long as we use `import type` syntax